Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -162,7 +162,7 @@ public async Task Execute_MainBranch()
         [TestMethod]
         public async Task Execute_PullRequest_SonarCloud()
         {
-            using var sut = new CacheProcessor(Mock.Of<ISonarWebService>(x => x.IsSonarCloud() == Task.FromResult(true)), CreateProcessedArgs("/k:key /d:sonar.pullrequest.base=master"), Mock.Of<IBuildSettings>(), logger);
```

Comment:
We simplified the `IsSonarCloud` and it doesn't need to by async. It doesn't call the server anymore.